src/App.js

```diff
@@ -1,14 +1,7 @@
-import './App.css';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-
-        Hello, React !
-      </header>
-    </div>
-  );
-}
+const App = () => (
```

Comment:
- [ ] Please make your `App.js` component a class-based component, since in this lesson you should learn to work with classes hence it is a requirement. [Screenshot from 2021-10-19 16-50-50](https://user-images.githubusercontent.com/46388113/137956167-cddb70ac-b43a-496b-b92c-4b9bdc16bea7.png)